libvirt/README.md

```diff
@@ -1,12 +1,11 @@
 # Machine Stats for Virtual environments (Virt-Stats)
 
-A simple and effective way to gather guest VM statistics (hostname, IP addresses) from a [libvirt](https://libvirt.org/)-based environment as a first layer of a [Tidal Migrations discovery process](https://guides.tidalmg.com/).
+A simple and effective way to gather guest VM statistics (hostname, IP addresses) from a [libvirt](https://libvirt.org/)-based environment as a first layer of a [Tidal Migrations discovery process](https://guides.tidalmg.com/). A common use case is to use this to integrate your KVM based virtual machine inventory into the Tidal Migrations Platform.
 
 ## Prerequisutes
 
 * [Python 3+](https://python.org/)
 * [libvirt](https://libvirt.org/) >=3.0.0 installed on both operator machine (i.e where you run Virt-Stats) and on remote machine (i.e where you run your virtual environmnent and guest VMs)
-* [QEMU guest agent](https://wiki.qemu.org/Features/GuestAgent) installed and running on guest VMs _(optional)_.
```

Comment:
I only removed this so I could comment.

If it is an optional prerequisite, why is it needed? prerequisites are not optional (by definition!). So I think we either include a description of what someone can and can't do if they do have this prerequisite installed. Or if there is no impact, then don't include it here.

Reply:
Totally agree. The only reason I left QEMU guest agent there is to remove that “optional” in the nearest future, since most of the facts we're going to add rely on QEMU guest agent, so the existence of the agent will become mandatory (at least for QEMU/KVM hypervisors).

Reply:
ok got it. so will keep it removed now, and well add it back when it provides utility to a user. and when we do, lets include what they get by having it and what they don't get if they don't have it.